package.json

```diff
@@ -30,36 +31,39 @@
   },
   "homepage": "https://github.com/babel/babel-library-boilerplate",
   "devDependencies": {
-    "babel": "^5.0.0",
-    "babelify": "^6.0.0",
-    "browserify": "^8.1.1",
-    "chai": "^2.0.0",
+    "babel": "^5.2.17",
```

Comment:
Some of these changes aren't actually doing anything. The `^` operator will pull in minor and patch releases. So, for instance, `^5.0.0` will pull in any version of Babel that is less than `^6.0.0`, including `^5.2.17`.